code/game/objects/items/storage/backpack.dm

```diff
@@ -350,6 +350,65 @@
 	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
 	STR.max_combined_w_class = 30
 
+/obj/item/storage/backpack/duffelbag/cursed
+	name = "living duffelbag"
+	desc = "A cursed clown duffelbag which hungers, maybe putting some food in it will stop it from eating you! Or maybe you can poison it this way..."
+	icon_state = "duffel-curse"
+	inhand_icon_state = "duffel-curse"
+	slowdown = 1.3
+	///counts time passed since it ate food
+	var/hunger = 0
+	///life left before it dies
+	var/hp = 100
```

Comment:
IMO, we should be using the bag's integrity for this, not an HP variable.

That way, poison can be used alongside other methods of dealing damage to clothing, like acid and fire.